static/docs/commands-reference/import-url.md

```diff
@@ -97,9 +97,13 @@ CLI tools for each service.
 
 When DVC inspects a DVC-file, its dependencies will be checked to see if any
 have changed. A changed dependency will appear in the `dvc status` report,
-indicating the need to reproduce this import stage. When DVC inspects an
-external dependency, it uses a method appropriate to that dependency to test its
-current status.
+indicating the need to reproduce this <abbr>import stage</abbr>. When DVC
+inspects an external dependency, it uses a method appropriate to that dependency
+to test its current status.
+
+Note that by default, import stages are locked in their DVC-files (via fields
+`lock: true` and `rev_lock`). Use `dvc update` manually on them to force
```

Comment:
rev_lock is not relevant for `import_url` I believe

Reply:
You're right. No lock at all in fact! I detected other changes in a Discord [chat](https://discordapp.com/channels/485586884165107732/563406153334128681/605183269436260352) with Ruslan. Updated in 0f4b94c

Reply:
I mean 4ffe63a